src/pages/saksbehandling/steg/formue/Formue.tsx

```diff
@@ -120,33 +119,16 @@ const Formue = (props: VilkårsvurderingBaseProps) => {
             lagreBehandlingsinformasjon({
                 sakId: props.sakId,
                 behandlingId: props.behandling.id,
-                behandlingsinformasjon: {
-                    formue: formueValues,
-                    ektefelle: RemoteData.isSuccess(eps)
-                        ? {
-                              fnr: eps.value.fnr,
-                              navn: eps.value.navn,
-                              kjønn: eps.value.kjønn,
-                              fødselsdato: eps.value.fødselsdato,
-                              alder: eps.value.alder,
-                              adressebeskyttelse: eps.value.adressebeskyttelse,
-                              skjermet: eps.value.skjermet,
-                          }
-                        : {
-                              // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-                              fnr: values.epsFnr!,
-                          },
-                    //Det skal kreves ny registerering av sats når EPS endres. Enten ved ny Fnr, eller fjerner/legger til EPS.
-                    //Denne sørger for at vi nuller ut sats steget hvis det er gjort en endring, og vi har en eksisterende bosituasjon
-                    bosituasjon:
-                        !erEktefelleUendret && eksisterendeBosituasjon
-                            ? {
-                                  ...eksisterendeBosituasjon,
-                                  delerBolig: null,
-                                  ektemakeEllerSamboerUførFlyktning: null,
-                              }
-                            : eksisterendeBosituasjon,
-                },
+                behandlingsinformasjon: { formue: formueValues },
+            })
+        );
+
+        // todo ai: wuuut to dooo here
+        await dispatch(
```

Comment:
hehe, vet ej om vi trenger å gøre någon speciell error handling knyttet till kallet for å lagre grunnlag for eps?

Reply:
Siden det er et API-kall så kan det jo feile. Det ryddigste er jo da å ta vare på resultatet av API-kallet og vise en feilmelding dersom det feiler.